products/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+
+class Collection(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'collections'
+
+class Gender(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'genders'
+
+class Product(models.Model):
+    product_name = models.CharField(max_length=45)
+    price        = models.IntegerField()
+    genders      = models.ForeignKey(Gender, on_delete=models.CASCADE, related_name="gender_products")
+    is_adult     = models.BooleanField(default=True)
+    collections  = models.ForeignKey(Collection, on_delete=models.CASCADE, related_name="collection_products")
+    categories   = models.ForeignKey(Category, on_delete=models.CASCADE, related_name="category_products")
+
+    class Meta:
+        db_table = 'products'
+
+class Color(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'colors'
+
+class Product_Color(models.Model):
```

Comment:
파이썬 클래스 네이밍 컨벤션은 `_`를 쓰지 않고 Camel Case로 작성하는 것 입니다!
```suggestion
class ProductColor(models.Model):
```